test/functional/api/window_spec.lua

```diff
@@ -363,39 +363,39 @@ describe('API/win', function()
     end)
   end)
 
-  describe('nvim_win_get_option, nvim_win_set_option', function()
+  describe('nvim_get_option_value(win), nvim_set_option_value(win)', function()
```

Comment:
There is already `API/win` in the description above, so I think `(win)` isn't needed
```suggestion
  describe('nvim_get_option_value, nvim_set_option_value', function()
```